README.md

```diff
@@ -1,11 +1,7 @@
-# lambda-template
-A GitHub template for quickly starting a new AWS lambda project.
+# lambda-sc-bucket-cleanup
+An application to clean up S3 buckets after it has been terminated from the Service Catalog.
```

Comment:
to clean up *an S3 bucket* after it has...

Reply:
fixed